cosmos/providers/dbt/render.py

```diff
@@ -111,12 +111,12 @@ def render_project(
 
         if "configs" in select:
             # TODO: coverme
-            if not set(select["configs"]).intersection(model.config.config_selectors):
+            if not set(select["configs"]).intersection(model.config.config_selectors) == set(select["configs"]):
                 continue
 
         if "configs" in exclude:
             # TODO: coverme
-            if set(exclude["configs"]).intersection(model.config.config_selectors):
+            if set(exclude["configs"]).intersection(model.config.config_selectors) == set(exclude["configs"]):
```

Comment:
this should be OR

Reply:
It feels like a great opportunity to add tests to cover this part of the code, which handn't been covered yet!